src/Components/BrowsePage/BrowsePage.scss

```diff
@@ -0,0 +1,133 @@
+        background-size: cover;
+        display: flex;
+        flex-direction: rows;
+        background-color: $dark;
+        justify-content: space-between;
+        margin: 0%;
+        padding: 1%;
+
+        .logo{
+          font-size: 9em;
+          font-weight: 400;
+          color: $mango;
+          height: 100%;
+        }
+    
+  }
+
+  .title-container{
+      height: 10vh;
+      margin: 0%;
+    
+      .title{
+          margin: 0%;
+        background-color: $dark;
+        color: $mango;
+        padding: 4%;
+        font-family: $Oswald;
+        font-size: 2.5em;
+      }
+  }
+
+  .grid-container{
+    margin: 0%;
+    padding: 5%;
+    height: 100%;
+
+    .inner-grid{
+        position: relative;
+        width: 100%;
+        margin: 2%;
+        padding: 0; 
+
+        .hexa{
+            border-radius: 25px;
+            list-style-type: none;
+            position: relative;
+            float: left;
+            width: 27.85714285714286%;
+            padding: 0 0 32.16760145166612% 0;
+            transform: rotate(-60deg) skewY(30deg);
+            overflow: hidden;
+            visibility: hidden;
+
+            .inner-hexagon{
+                position: absolute;
+                top: 0;
+                left: 0;
+                width: 100%;
+                height: 100%;
+                background: $mango;
+                transform: skewY(-30deg) rotate(60deg);
+                visibility: visible;
+                
+                .hexa-image{
+                    border-radius: 25px;
+                    width: 55%;
+                    margin: 20%;
+                    height: 40%;
+                
+                }
+
+                .title{
+                    font-size: 1.3em;
+                    font-family: 'Comfortaa', cursive; 
+                    color: $darkPurple;
+                    margin: -10%
+                }
+
+                .large-title{
+                    color: $darkPurple;
+                    font-family: 'Comfortaa', cursive;
+                    margin: -10%;
+                    font-size: 70%;
+                    overflow: auto;
+                }
+            }
+            .inner-hexagon:hover{
+                background-color: darken($mango, 20%);
+            }
+        }
+                
+
+        .hexa:nth-child(3n+2){
+            margin: 0 1%;
+        }
+
+        .hexa:nth-child(6n+4), .hexa:nth-child(6n+5), .hexa:nth-child(6n+6) {
+            margin-top: -6.9285714285%;
+            margin-bottom: -6.9285714285%;
+            transform: translateX(50%) rotate(-60deg) skewY(30deg);
+        }
+
+        .hexa:nth-child(6n+4) { 
+            margin-left: 0.5%;
+        }
+
+        .hexa:hover{
+            background-color: yellow;
+        }
+    }
+
+    .inner-grid:after {
+        content: "";
+        display: block;
+        clear: both;
+    }
+
+
+  }
+
+
+}
```

Comment:
I will do more research about this and will get back to you with more information.